app/models/ability.rb

```diff
@@ -0,0 +1,35 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user (not logged in)
+
+    # TODO: currently checks if it has the class, in the future it will check
+    # a `roles` attribute or similar
+
+    # TODO: using `voter_id`, `artist_id` which will become `user_id` in future
+    # after models are unified
+
+    # TODO: why do admins not require activation?
```

Comment:
Admins currently can only be created by another admin using the Portal, so they don't need activation.  But that's a side effect of the implementation, not a design choice.  In the future I still wouldn't want randos to be able to try to sign up as an admin, but their initial signup should go through the same flow as everyone else.